platform/entanglement/src/services/CopyTask.js

```diff
@@ -185,6 +203,10 @@ define(
         CopyTask.prototype.perform = function(){
             this.deferred = this.$q.defer();
 
+            if (!this.parent.hasCapability('composition')){
```

Comment:
Out of curiosity, are there cases where this situation happens? I would think that `CopyService.validate` would prevent that